### ISSUE TITLE
"No translation" build contains all the translations

### ISSUE THREAD

**Issue by mekedron** [kiprotect/klaro#515]
**Steps to reproduce**

Download latest version via NPM

**Current behavior**

Translations are still included even though this is a build with no translations included.

<img width="1849" alt="image" src="https://github.com/user-attachments/assets/e10378a3-bc4b-4d34-8837-3220501fbed5">

**Expected behavior**

Translations must not be included.

**Comment by kanow** [comment]
Yes, it's also there in the files in dist folder  here in github. Should be fixed there too.

**Comment by darkopetreski** [comment]
Any update on this, when will be fixed ?

**Comment by tdonohue** [comment]
We are hitting this same bug with version 0.7.21 and 0.7.22.  It appears that there is no longer a way to use Klaro without all the embedded translations.  `klaro-no-translations.js` appears to be the same as `klaro.js`.
